ckanext/dcat/profiles.py

```diff
@@ -471,15 +475,19 @@ def _add_triple_from_dict(self, _dict, subject, predicate, key,
                               fallbacks=None,
                               list_value=False,
                               date_value=False,
+                              value_modifier=None,
```

Comment:
I would add the new parameter at the end, maybe someone is already using `_add_triple_from_dict` in their code, so this would break it.